src/index.js

```diff
@@ -8,6 +8,7 @@ const loggerAlias: Logger = logger
 type Options = {
   cwd?: string,
   async?: boolean,
+  input?: string,
```

Comment:
One of the assumptions of `run` command is to provide the same options for `sync` and `async` calls. This breaks that assumption as `input` can be used only for `async` call which will be less clear for the users and a bit more confusing.

Also do we really need it ? Can user just provide input through `stdio` ?

Reply:
Thanks for the reply. I read the Node.js docs for `child_process` and couldn't figure out from them how to pass in a "stdio" value using a decent syntax, which out using a shell pipe (|). Do you have an example?

The current approach could be updated to address your concerns in two ways:

 1. `input` could be renamed to `syncInput`, making it clear it is only for `sync` calls.
 2. Attempting combine a `syncInput` argument with `async: true` would throw an error, notifying the development immediately if they still managed to combine the two.